docs/man_pages/project/configuration/generate-splash.md

```diff
@@ -0,0 +1,34 @@
+generate splashscreen
```

Comment:
An alternative is to go with `generate splash` instead of `generate splashscreen` for the command name.